### ISSUE TITLE
Parallel Batch Processing

### ISSUE THREAD

**Issue by david-hoffman** [ilastik/ilastik#1995]
## Description

When batch processing, give the option to process each file in parallel (great for processing many small images).

**Comment by k-dominik** [comment]
that would be nice to have, I agree. Although the classification will be parallel regardless. Feature computation will be limited to a single thread, currently, when using small images.
